test/net/sourceforge/kolmafia/session/EquipmentManagerTest.java

```diff
@@ -180,4 +188,33 @@ public void canParseStatus() {
       assertItemUnequip(Slot.FOLDER5);
     }
   }
+
+  @Nested
+  class professor {
+    @ParameterizedTest
+    @CsvSource({
+      "mafia thumb ring, false",
+      "Treads of Loathing, false",
+      "panhandle panhandling hat, false",
+      "batskin belt, true",
+      "mafia wedding ring, true"
+    })
+    public void itShouldEquipWhatWasRequestedForProf(String item, String equippable) {
+      AdventureResult itemAR = ItemPool.get(item);
+      boolean canBeEquipped = Boolean.parseBoolean(equippable);
+      var cleanups =
+          new Cleanups(
+              withPath(AscensionPath.Path.WEREPROFESSOR),
+              withIntrinsicEffect(EffectPool.MILD_MANNERED_PROFESSOR),
+              withStats(1, 5, 1),
+              withItem(itemAR));
+      try (cleanups) {
+        assertEquals(1, KoLCharacter.getBaseMoxie());
+        assertEquals(5, KoLCharacter.getBaseMysticality());
+        assertEquals(1, KoLCharacter.getBaseMuscle());
+        assertTrue(InventoryManager.hasItem(itemAR));
```

Comment:
these are testing the cleanups work correctly -- this might have been useful when writing the test, but I don't think it needs to be present in the final form.

Reply:
Done.  I decided to explicitly set stats because withLevel does not seem to accommodate that a Level 1 Professor has mysticality of 5.  I also decided checking for the item was testing cleanups stopped it.  Thank you.